MANIFEST.in

```diff
@@ -10,10 +11,10 @@ include src/cpp/pybind11/CMakeLists.txt
 include src/cpp/pybind11/LICENSE
 include src/cpp/pybind11/tools/*
 include src/cpp/_pythia8.cpp
-recursive-include src/cpp/pythia8307/include *
-recursive-include src/cpp/pythia8307/src *
-include src/cpp/pythia8307/COPYING
-include src/cpp/pythia8307/AUTHORS
+recursive-include src/cpp/pythia83/include *
+recursive-include src/cpp/pythia83/src *
+include src/cpp/pythia83/COPYING
+include src/cpp/pythia83/AUTHORS
```

Comment:
I think you need another line for src/cpp/private_access.hpp

Reply:
Add it.